detection_rules/main.py

```diff
@@ -92,14 +92,17 @@ def generate_rules_index(ctx: click.Context, query, overwrite, save_files=True):
 @root.command('import-rules')
 @click.argument('input-file', type=click.Path(dir_okay=False, exists=True), nargs=-1, required=False)
 @click.option('--directory', '-d', type=click.Path(file_okay=False, exists=True), help='Load files from a directory')
-def import_rules(input_file, directory):
+@click.option('--ignore-invalid-files', '-i', is_flag=True, help='Ignore files with invalid rule data')
```

Comment:
with this [comment](https://github.com/elastic/detection-rules/pull/3167/files#r1355133660), you should be able to revert this whole file